pkg/util/k8sutil/tpr_util.go

```diff
@@ -75,8 +75,8 @@ func (c *TfJobRestClient) Client() *http.Client {
 }
 
 func (c *TfJobRestClient) Watch(host, ns string, httpClient *http.Client, resourceVersion string) (*http.Response, error) {
-	return c.restcli.Client.Get(fmt.Sprintf("%s/apis/%s/%s/namespaces/%s/%s?watch=true&resourceVersion=%s",
-		host, spec.CRDGroup, spec.CRDVersion, ns, spec.CRDKindPlural, resourceVersion))
+	return c.restcli.Client.Get(fmt.Sprintf("%s/apis/%s/%s/%s?watch=true&resourceVersion=%s",
```

Comment:
Why don't we need namespace anymore?

Reply:
In k8s read api(such as watch,list), http request URL with no namespace implies that the request will look up the target object in all namespaces.
See https://github.com/kubernetes/client-go/issues/159. 
You can also try a test using commander below:
          kubectl --alsologtostderr -v=8 get job -w --all-namespace

Reply:
Thanks for the explanation.